backbone/src/main/java/org/researchstack/backbone/ui/step/layout/SurveyStepLayout.java

```diff
@@ -153,12 +154,14 @@ public void onClick(View view)
         });
 
         if (questionStep != null) {
-            if (!TextUtils.isEmpty(questionStep.getTitle())) {
+            if (questionStep.getStepBodyClass() == FormBody.class
+                    && !TextUtils.isEmpty(questionStep.getTitle())) {
```

Comment:
why the question affects the title here? they both need to be supported, and they are not related